pages/wiki/porting-guide.md

```diff
@@ -20,76 +20,76 @@ layout: documentation
-<p>Start by cloning the most appropriate layer into an src/meta-XXX-hybris/ directory in your AsteroidOS build tree. From now on, XXX will be a reference to&nbsp;your watch’s codename. Once you have that layer setup, <a href="https://github.com/AsteroidOS/meta-swift-hybris/commit/b665878f80936f21cec1757cdc4d10fa95af1841">roughly&nbsp;replace</a> all references to the previous codename&nbsp;with XXX in the layer and add the path to your layer in your build/conf/bblayers.conf file.</p>
-<p>The&nbsp;layer should describe&nbsp;a machine configuration to bitbake in&nbsp;<a href="https://github.com/AsteroidOS/meta-dory-hybris/blob/master/conf/machine/dory.conf">conf/machine/XXX.conf</a>. The definition of a machine allows to <a href="https://github.com/AsteroidOS/meta-dory-hybris/blob/master/conf/machine/dory.conf#L25">install hardware-specific packages</a>&nbsp;in a generated rootfs and&nbsp;to give very general information about the hardware adaptation like the <a href="https://github.com/AsteroidOS/meta-dory-hybris/blob/master/conf/machine/dory.conf#L18">kernel</a> or <a href="https://github.com/AsteroidOS/meta-dory-hybris/blob/master/conf/machine/dory.conf#L10">egl providers</a>&nbsp;that should be used. Having a new machine also allows to modify other layers’&nbsp;recipes based on the build’s target. For example a .bbappend file using the _XXX suffix appropriately can&nbsp;<a href="https://github.com/AsteroidOS/meta-dory-hybris/blob/master/recipes-android/brcm-patchram-plus/brcm-patchram-plus_git.bbappend#L1">modify built files</a>, <a href="https://github.com/AsteroidOS/meta-dory-hybris/blob/master/recipes-qt/qt5/qt5-qpa-hwcomposer-plugin_git.bbappend#L1">apply&nbsp;patches</a>,&nbsp;<a href="https://github.com/AsteroidOS/meta-dory-hybris/blob/master/recipes-nemomobile/sensorfw/sensorfw_git.bbappend#L1">add build dependencies</a>, <a href="https://github.com/AsteroidOS/meta-dory-hybris/blob/master/recipes-multimedia/pulseaudio/pulseaudio_%25.bbappend#L1">add runtime dependencies</a> or&nbsp;<a href="https://github.com/AsteroidOS/meta-dory-hybris/blob/master/recipes-android/brcm-patchram-plus/brcm-patchram-plus_git.bbappend#L3">change compilation flags</a>. bbappends are very powerful but&nbsp;should be used scarcely to&nbsp;avoid conflicts with other layers.</p>
+<p>Start by cloning the most appropriate layer into an <code>src/meta-XXX-hybris/</code> directory in your AsteroidOS build tree. From now on, XXX will be a reference to&nbsp;your watch’s codename. Once you have that layer setup, <a href="https://github.com/AsteroidOS/meta-swift-hybris/commit/b665878f80936f21cec1757cdc4d10fa95af1841">roughly&nbsp;replace</a> all references to the previous codename&nbsp;with XXX in the layer and add the path to your layer in your <code>build/conf/bblayers.conf</code> file.</p>
```

Comment:
Now I see why you were asking about the `&nbsp;` on Matrix.  All of these on this page should be deleted.

Reply:
Ok and the other pages?

Reply:
A quick check shows that they probably should be removed in all pages.  For just the `useful-commands.md` file, each time it's used, it is just before a `####` heading marker, but the better way to create extra spacing is to omit the `&nbsp;` and use a `###` heading marker instead.